src/Utilities/Utilities.php

```diff
@@ -0,0 +1,60 @@
+<?php
+/**
+ * Pinterest for WooCommerce Utilities.
+ *
+ * @package     Pinterest_For_WooCommerce/Classes/
+ * @since       x.x.x
+ */
+
+declare( strict_types=1 );
+
+namespace Automattic\WooCommerce\Pinterest\Utilities;
+
+use \Error;
+
+/**
+ * Utilities class.
+ *
+ * @since x.x.x
+ */
+class Utilities {
+
+	const PINTEREST_ACCOUNT_CONNECTION_TIMESTAMP = PINTEREST_FOR_WOOCOMMERCE_PREFIX . '-account-connection-timestamp';
+
+	/**
+	 * Set the account connection timestamp.
+	 *
+	 * @since x.x.x
+	 * @throws Error Wrong value provided for the timestamp.
+	 * @param int $timestamp Timestamp to set as the account connection moment.
+	 * @return int
+	 */
+	public static function set_account_connection_timestamp( $timestamp = null ): int {
+		if ( is_null( $timestamp ) ) {
+			$timestamp = time();
+		}
+
+		if ( ! is_int( $timestamp ) ) {
```

Comment:
Do you think there's ever a time where we might want to specify the timestamp instead of it just being the current moment? Right now when this is called, the timestamp is not specified. This makes me think that maybe we don't need the `$timestamp` parameter at all and we can just call `time()` directly. That would also have the additional benefit of not needing these checks of the passed parameter.

If you do feel that keeping the parameter is necessary, then I think it would make sense to declare it as a nullable type, e.g.:

```php
	public static function set_account_connection_timestamp( ?int $timestamp = null ): int {
```

Then we only need to verify if it's `null`, and don't need to otherwise type-check the parameter.

Reply:
Removing the param. We can always add it later if it will turn out to be necessary.